### PR TITLE
14 feature header

### PR DESCRIPTION
**Pull request by Koji47**
Header component added

### CODE REVIEW

File: src/components/Header/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+              controls
+              data-testid="video"
+            ></video>
+            <div className="header__video-container--lower-half"></div>
+          </div>
+        </div>
+      )}
+    </div>
+  );
+};
```
Comment (olirob93) [inline src/components/Header/Header.tsx:66]: Just looking at the disgns there is a bit more space between the text and either the video or image. Take a look at seeing if you can make it look slightly better.

**Review by olirob93 — CHANGES_REQUESTED** [review]
this is really good just a small thing really. Great you've got some tests in there!

File: src/components/Header/Header.scss [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    display: flex;
+    justify-content: center;
+  }
+  &__video-container {
+    height: fit-content;
+    padding: 0;
+    width: auto;
+  }
+
+  &__video-container--upper-half {
```
Comment (Charlie-robin) [inline src/components/Header/Header.scss:92]: Can you add the modifiers with the element?

```scss
 &__video-container {
    // SOME STYLES HERE...
    &--upper-half {
      // SOME STYLES HERE...
    }
  }
```

File: src/components/Header/Header.scss [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+  &__video-positioning {
+    display: inline;
+  }
+
+  &__location {
+    color: $color-white;
+    padding-left: 2rem;
+    font-family: $modern-content;
+    &--venue {
```
Comment (Charlie-robin) [inline src/components/Header/Header.scss:110]: Like you have done here 👍

**Review by Charlie-robin — CHANGES_REQUESTED** [review]
There are some little tweaks to make to follow conventions.

Can you also update your branch and add some information about your component to the readme as it will be used by multiple pages.

**Review by olirob93 — APPROVED** [review]
looks good

File: src/components/Header/Header.tsx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+  videoUrl,
+  locationVenue,
+  locationCity,
+}: HeaderProps) => {
+  return (
+    <div className="header">
+      <div className="header__content">
+        <h4 className="header__brand">DOT TO DOT - LOCAL MAKERS</h4>
+        <h1 className="header__title">{title}</h1>
+        <p className="header__subtitle">{subTitle}</p>
```
[the remark this answers is not in the document]
Comment (Koji47) [reply src/components/Header/Header.tsx]: all changes have been made now, ready for final review :)

**Review by Charlie-robin — APPROVED** [review]

**Pull request merged**